content/downloads.md

```diff
@@ -0,0 +1,10 @@
+---
+date: 2014-03-10
+title: Download
+image: 'speakers.jpg'
+---
+[Install libvarnam](../post/install-libvarnam)
+
+[Install iBus engine for varnam](../post/install-ibusvarnam)
+
+[Install Varnam Daemon](../post/install-varnamd)
```

Comment:
We need the main website to be clear for the common users. Varnamd is not needed.

We need to make the general site as non-technical as possible for common folks to easily understand, download & install